### PR TITLE
Update dev deps: cleanup folder

### PR DESCRIPTION
**Pull request by chrisnojima**
- [x] darwin
- [x] ios
- [x] android
- [x] Cleanup folders outside of widget view

# Notes
Webpack > 4.8 is super messed up. It's treeshaking is doing something I can't figure out and there's no indicators why its doing this so we can't upgrade this in the short term (i tried 4.10 and 4.11 which just came out). https://github.com/webpack/webpack/issues/7425 . I tried a bunch of things but it still was acting weird and isn't worth the risk until this ticket is closed

### CODE REVIEW

**Comment by chrisnojima** [comment]
@keybase/react-hackers this is ready

File: .eslintrc [hunk 1]
```diff
@@ -35,7 +35,7 @@
         "flowtype/delimiter-dangle": ["error", "always-multiline"],
         "flowtype/generic-spacing": "off",
         "flowtype/no-dupe-keys": "off",
-        "flowtype/object-type-delimiter": ["error", "comma"],
+        "flowtype/object-type-delimiter": "off",
```
Comment (akalin-keybase) [inline .eslintrc]: why was this needed?
